src/groovy/org/transmartproject/db/dataquery/highdim/HighDimensionResourceService.groovy

```diff
@@ -89,13 +89,11 @@ class HighDimensionResourceService implements HighDimensionResource {
 
         HashMultimap multiMap = HashMultimap.create()
```

Comment:
These changes enable multiple modules to be applicable for the same 'platform'. I.e., these modules query different datatypes (subject data, SNPs), but these datatypes may be related to the same platform, e.g., SNP data.